### PR TITLE
Feature detection for customElements v1

### PR DESCRIPTION
**Pull request by JamesLMilner**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,69 +1,74 @@
...
-    this.webShare = 'share' in navigator;
-    if (this.webShare) {
-      this.text = document.createTextNode(this.getAttribute('text') || 'Share');
-      this.button = document.createElement('button');
-      this.button.appendChild(this.text);
-      this.innerHTML = '';
-      this.appendChild(this.button);
-    }
-  }
+if ('registerElement' in document) {
```
Comment (philnash) [inline src/index.js:2]: [`registerElement` is deprecated](https://developer.mozilla.org/en-US/docs/Web/API/Document/registerElement). Will update this to check for `window.customElements`.

**Comment by philnash** [comment]
Cheers James, this is a good start towards this being less useless!

File: src/index.js (hunk 1 above)
Comment (JamesLMilner) [reply src/index.js:2]: Oops... Not the best feature detection then ;)

Comment (philnash) [reply src/index.js:2]: Better than my initial attempts at feature detection at least!

**Pull request merged**